server/neptune/workflows/internal/pr/receiver/shutdown.go

```diff
@@ -0,0 +1,44 @@
+package receiver
+
+import (
+	"github.com/runatlantis/atlantis/server/neptune/workflows/internal/deploy/request"
+	workflowMetrics "github.com/runatlantis/atlantis/server/neptune/workflows/internal/metrics"
+	"go.temporal.io/sdk/temporal"
+	"go.temporal.io/sdk/workflow"
+)
+
+const (
+	ShutdownSignalID = "pr-close"
+)
+
+type ShutdownReceiver struct {
```

Comment:
Also not sure why this needs to be in its own struct, there's like no state being maintained here, just call the logic in the function itself (or create a new function) in the runner file.